packages/@aws-cdk/aws-elasticloadbalancingv2/test/integ.alb3.ts

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env node
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as cdk from '@aws-cdk/core';
+import * as elbv2 from '../lib';
+
+/* Credit to @TrueBrain and @misterjoshua for the IPv6 workaround */
```

Comment:
is there an issue or comment this can point to instead

Reply:
#894 

Seems like an oldy but I couldn't find another method of implementation with existing cdk packages. Should I just reference it with URL?